### PR TITLE
added docstrings to visualization and interpretation

### LINKED ISSUE #12: add docstrings to `visualize.py` and `interpretation.py`

### LINKED ISSUE #16: change input parameter name from `input`
`input` is a built-in function in Python, so we should change the name of this variable (see https://docs.python.org/3/library/functions.html#input).

I suggest `input` -> `x`, here:
https://github.com/lowe-lab-ucl/cellx-predict/blob/b5917e3f3fd6c1fe124eff9c972c553e839cc44d/cellxpredict/visualization.py#L7

and here:
https://github.com/lowe-lab-ucl/cellx-predict/blob/b5917e3f3fd6c1fe124eff9c972c553e839cc44d/cellxpredict/visualization.py#L31

### PR DESCRIPTION
**Pull request by chris-soelistyo**

### CODE REVIEW

File: cellxpredict/interpretation.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ def latent_space_traversal(
     output_grid: np.ndarray
         - An array representing the 2D collage of images generated by the
```
Comment (quantumjot) [inline cellxpredict/interpretation.py]:
```suggestion
        An array representing the 2D collage of images generated by the
```

Comment (quantumjot) [reply cellxpredict/interpretation.py]: Could you remove these dashes in the rest of the docs too?

File: cellxpredict/visualization.py [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
 import numpy as np
 from skimage.filters import median as median_filter
 from skimage.morphology import disk
 
-from typing import List
 
 def float_to_uint8(input: np.ndarray[np.float32]) -> np.ndarray[np.uint8]:
-    '''
+    """
     Transforms an np.float32 image into a np.uint8 image
```
Comment (quantumjot) [inline cellxpredict/visualization.py]:
```suggestion
    """Transforms an np.float32 image into a np.uint8 image
```

No newline after the triple quotes

File: cellxpredict/visualization.py [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
     mn = input.min()
     mx = input.max()
     mx -= mn
-    output = ((input - mn)/mx) * 255
+    output = ((input - mn) / mx) * 255
     return output.astype(np.uint8)
 
+
 def norm_channel(
     input: np.ndarray,
```
Comment (quantumjot) [inline cellxpredict/visualization.py]:
```suggestion
    x: np.ndarray,
```

`input` is a protected function so we should change this to another parameter, e.g. `x` - see #16

**Review by quantumjot — CHANGES_REQUESTED** [review]
Great stuff! There are a few small changes to make (suggested below).

**Comment by chris-soelistyo** [comment]
Have sorted through the comments, let me know if it looks good! I also added some extra `args` to the `isort` pre-commit in the `yaml` file because it seemed like `isort` and `black` were conflicting on the `models.py` file, let me know if this seems right.

**Review by quantumjot — APPROVED** [review]
Looks great!

**Pull request merged**